demo-ehr-app/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```

Comment:
Shall we have a proper title here?

Reply:
Fixed. https://github.com/wso2/solutions-healthcare-demos/pull/2/commits/afd92c12223917bcd39b551d212f4129aae9b4bc